lms/static/js/learner_dashboard/views/entitlement_unenroll_view.js

```diff
@@ -0,0 +1,158 @@
+(function(define) {
+    'use strict';
+    define(['backbone',
+        'jquery',
+        'underscore',
```

Comment:
Doesn't look like you actually use underscore in this file.  Shouldn't need to import it (and it's a deprecated tech for us).

Reply:
Removed.